proxy/logging/LogAccess.cc

```diff
@@ -1144,7 +1144,7 @@ void
 LogAccess::set_client_req_url(char *buf, int len)
 {
   if (buf) {
-    m_client_req_url_len = len;
+    m_client_req_url_len = (m_client_req_url_len <= len) ? m_client_req_url_len : len;
```

Comment:
Stylistically, this might be better as `std::min(m_client_req_url_len, len)`.

Reply:
yeah, makes sense. Although, the method sets `m_client_req_url_len` to the new length (possibly, the setter gets called multiple times?), so I'll go with your first suggestion and leave the "+1" (for strlcpy) explicit.

```
m_client_req_url_len = std::min(m_client_req_url_len, len);

```